src/components/SearchAndSort/SearchAndSort.js

```diff
@@ -725,28 +726,34 @@ export class SearchAndSort extends Component {
   };
 
   render() {
+    const { finishedResourceName } = this.props;
+
+    const isMappingProfile = finishedResourceName === ENTITY_KEYS.MAPPING_PROFILES;
```

Comment:
Maybe it makes sense to not get tied to the `Mapping profiles` entity and pass some prop like 'isDetailsPaneFullScreen' from the root (`<MappingProfiles>`)? What do you think?